core/trino-server-core/README.txt

```diff
@@ -0,0 +1,5 @@
+Trino is a distributed SQL query engine.
```

Comment:
This README is not very informative

Reply:
Expanded a bit .. but its supposed to be small..